### ISSUE TITLE
Chat - Green line is not displayed

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#43148]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.79-2
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):** monitozb87+admin2305@gmail.com
**Issue reported by:** Applause - Internal Team

**Issue found when executing PR** https://github.com/Expensify/App/pull/42568
## Action Performed:
1. Open the native app
2. User B send message to User A
3. Chat between users marked as unread (for user A)
4. User A navigates to chat with user B
5. User B send another message

## Expected Result:
Verify that after step 4, user A sees a new message marker with a green line above first message from user B. The new marker line is displayed over the original unread message and Verify that after step 5, the new marker line remains, as messages come through. It won’t go away unless user switches to another chat.

## Actual Result:
Green line is not displayed

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [x] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/8feaefeb-3831-4893-b037-33673c6d37d7

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by muttmuure** [comment]
Low priority

**Comment by jacobkim9881** [comment]
Can this issue be reopened? I want to take care of this issue.
